demisto_sdk/commands/common/hook_validations/script.py

```diff
@@ -273,3 +275,75 @@ def is_valid_script_file_path(self) -> bool:
                     return False
 
         return True
+
+    def is_there_separators_in_names(self) -> bool:
+        """
+        Check if there are separators in the script folder or files.
+
+        Returns:
+            true if the folder/files names are valid and there are no separators, and false if not.
+        """
+        is_unified_script = self.current_file.get('script', '') not in ['-', '']
+
+        if is_unified_script:
+            return True
+
+        answers = [
+            self.check_separators_in_folder(),
+            self.check_separators_in_files()
+        ]
+
+        return all(answers)
+
+    def check_separators_in_folder(self) -> bool:
+        """
+        Check if there are separators in the script folder name.
+
+        Returns:
+            true if the name is valid and there are no separators, and false if not.
+        """
+
+        script_folder_name = self.file_path.rsplit('/', 2)[1]
```

Comment:
```suggestion
        script_folder_name = os.path.basename(os.path.dirname(self.file_path))
```
this will make it work on windows as well.